target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -36,8 +36,10 @@ edgecore,oap100)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "blue:wlan2g" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "blue:wlan5g" "phy1tpt"
 	;;
-engenius,eap1300)
-	ucidef_set_led_netdev "lan" "LAN" "blue:lan" "eth0"
+engenius,eap1300 |\
+engenius,eap1300ext)
+	ucidef_set_led_default "power" "POWER" "orange:power" "1"
```

Comment:
Feels to me like this could be done in the dts instead which is always more elegant.
`default-state` property (set to "on") if it should be always on (I'm not sure if that's your intention)

But maybe you were just looking for `function = LED_FUNCTION_POWER;`
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_asus_rt-ax53u.dts

I'm fairly new to development here, so take this with a grain of salt and maybe wait for other responses on this matter :)

Reply:
The power LED was already supported, this just makes it show up in the config/luci by default. If this is something that can automatically be picked up if it's set properly in the DTS, then it seems like all the LEDs should be converted. Having configuration split up would be annoying.

Reply:
FWIW, I have no strong opinions on how to do this, I just want to get these re-supported so that whenever 23.x comes out I don't have to be patching stuff.

Reply:
and I'm curious how this is actually handled, because I don't know it, yet. :)
If ucidef is required for it to show up in luci, then this should be just fine and the right way to do it. But I see only one mikrotik ap that even defines a power led like this for the whole target.

Reply:
Yeah, IDK, I like to set the power led trigger to heartbeat, and I was annoyed at having to add the LED in luci to do so.

Reply:
If it can be set via DTS, then it should be set via DTS, especially since its the power LED.

Reply:
I commented on this upthread, and the pull request no longer contains the code.

Reply:
@robimarko @Djfe I have confirmed that luci won't show the LEDs with DTS alone, do you still request I remove this change?

Reply:
I don't agree with the logic here, but this change is also not really related to anything else being done, so have removed it.

The power LED is configured by DTS.